test/with_dummyserver/test_socketlevel.py

```diff
@@ -2380,7 +2380,7 @@ def body_generator() -> typing.Generator[bytes, None, None]:
             body.seek(0, 0)
             should_be_chunked = True
         elif body_type == "file_text":
-            body = io.StringIO("x" * 10)
+            body = io.StringIO("x\x80\x81")
```

Comment:
We want all chunked bodies to be encoded to UTF-8, not only StringIO. Can you please use `x * 9 + \x80` everywhere?